config/fixes/convention-eccodes.yaml

```diff
@@ -179,28 +186,30 @@ convention_name:
           - 235130 # avg_t
           - t
           - avg_t
-          - ta
           - mt # legacy eccodes
           - T # AQUA ERA5
+          - ta # cmor
         grib: 130
       tcc: # Total cloud cover
         source:
           - 164 # original GRIB2 code
           - 228006 # meantcc in 0-1 GRIB2 code
           - 228164 # avg_tcc in %
+          - 131164 # ICON used this at some point
           - tcc
-          - clt # cmor
           - meantcc #legacy eccodes name
           - avg_tcc
           - TCC # AQUA ERA5
-        grib: 131164
+          - clt # cmor
+        grib: 228164
       tciw: # Total column vertically-integrated cloud ice water
         source:
           - 79 # original GRIB2 code
           - 235088 # avg corresponding to 79
           - tciw
           - avg_tciw
           - clivi # vertically integrated cloud ice (https://easy.gems.dkrz.de/DYAMOND/NextGEMS/Models/c2_icon_ngc2013_output.html)
+          - iwp
```

Comment:
I guess this is not cmor either, but where is coming from?

Reply:
again, ece4 invention. Sorry I misse dthese, it was actually my plan to keep these in a separate fixes file

Reply:
Ok, for now I removed them. If instead you think it is ok to have also EC-Earth4 'inventions' in here I will put them back